crates/turborepo-lib/src/task_hash.rs

```diff
@@ -477,6 +477,9 @@ impl<'a> TaskHasher<'a> {
                         "COLORTERM",
                         "TERM",
                         "TERM_PROGRAM",
+                        // Jetbrains IDE
+                        "JB_IDE_HOST",
+                        "JB_IDE_PORT",
```

Comment:
We might need more based on [this comment](https://github.com/containers/toolbox/issues/618#issuecomment-1858311856)
```suggestion
                        // Jetbrains IDE
                        "JB_IDE_*",
                        "JB_INTERPRETER",
```